training/distributed_training/pytorch/data_parallel/efficientnet/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG region
+
+# Download base PT DLC
```

Comment:
i think we need to put a comment here to highlight the minimum base DLC required for this example. (and why)

Reply:
Sure thing. I will add it